core/src/main/proto/wallet.proto

```diff
@@ -434,6 +434,7 @@ message Wallet {
     UNENCRYPTED = 1;                 // All keys in the wallet are unencrypted
     ENCRYPTED_SCRYPT_AES = 2;        // All keys are encrypted with a passphrase based KDF of scrypt and AES encryption
     ENCRYPTED_BLS_KEYEXCHANGE_AES = 3;           // Not used for wallet
+    ENCRYPTED_ECDH_KEYEXCHANGE_AES = 4;          // Not used for wallet
```

Comment:
This change requires that the Protos.java be regenerated by protoc.

Reply:
There are many changes in `Protos.java` probably due to an updated `protoc` that actually matches the `protobuf` library used by `dashj`.

Reply:
This probably means that the other protobuf generated files in dashj are using an out of date protobuf compiler